api/tests/unit/application/preHandlers/certification-session-authorization_test.js

```diff
@@ -0,0 +1,55 @@
+const { expect, sinon, domainBuilder, catchErr } = require('../../../test-helper');
+const { NotFoundError } = require('../../../../lib/application/http-errors');
+const CertificationSessionAuthorization = require('../../../../lib/application/preHandlers/certification-session-authorization');
+const sessionAuthorizationService = require('../../../../lib/domain/services/session-authorization-service');
+const certificationCourseRepository = require('../../../../lib/infrastructure/repositories/certification-course-repository');
+
+describe('Unit | Pre-handler | Certification Session Authorization', function() {
+
+  describe('#verify', function() {
+    let userId, certificationCourse, request;
+
+    beforeEach(function() {
+      userId = 1;
+      certificationCourse = domainBuilder.buildCertificationCourse();
+      request = {
+        auth: { credentials: { accessToken: 'valid.access.token', userId } },
+        params: {
+          id: certificationCourse.id,
+        },
+      };
+      sinon.stub(sessionAuthorizationService, 'isAuthorizedToAccessSession');
+      sinon.stub(certificationCourseRepository, 'get');
+    });
+
+    context('when user has access to session', function() {
+
+      it('should reply with true', async function() {
```

Comment:
**Suggestion**: When user is allowed to access the session -> should return true